tests/unit-tests/rest-api/test-class-sensei-rest-api-course-actions-controller.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * Test for Sensei_REST_API_Course_Actions_Controller.
+ *
+ * @covers Sensei_REST_API_Course_Actions_Controller
+ */
+class Sensei_REST_API_Course_Actions_Controller_Test extends WP_UnitTestCase {
+	use Sensei_Test_Login_Helpers;
+
+	/**
+	 * A server instance that we use in tests to dispatch requests.
+	 *
+	 * @var WP_REST_Server $server
+	 */
+	protected $server;
+
+	/**
+	 * Sensei post factory.
+	 *
+	 * @var Sensei_Factory
+	 */
+	protected $factory;
+
+	/**
+	 * Test specific setup.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		global $wp_rest_server;
+		$wp_rest_server = new WP_REST_Server();
+		$this->server   = $wp_rest_server;
+
+		do_action( 'rest_api_init' );
+
+		$this->factory = new Sensei_Factory();
+	}
+
+	public function tearDown() {
+		parent::tearDown();
+		$this->factory->tearDown();
+	}
+
+	public function testAddUsersToCourses_RequestsGiven_ReturnsSuccessfulResponse() {
```

Comment:
Naming convention doesn't work well for REST API tests (as they are integration ones).
However, describing all three parts (what we test, state, and expected result) still sounds good.